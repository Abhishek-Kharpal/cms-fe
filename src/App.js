import { BrowserRouter, Route, Routes } from 'react-router-dom';
import { LOGIN_ROUTE, ERROR_ROUTE, REGISTER_ROUTE, HOME_ROUTE } from './constants/routes';
import Login from './pages/login';
import Register from './pages/register';
import Home from './pages/home';
import PageNotFound from './pages/pageNotFound';
import Error from './pages/error';

function App() {
  return (
    <div>
      <BrowserRouter>
        <Routes>
          <Route path={LOGIN_ROUTE} element={<Login />} />
          <Route path={REGISTER_ROUTE} element={<Register />} />
          <Route path={HOME_ROUTE} element={<Home />} />
          <Route path={`${ERROR_ROUTE}/:errorCode?`} element={<Error />} />
          <Route path='*' element={<PageNotFound />} />
        </Routes>
      </BrowserRouter>
    </div>
  );
}

export default App;
