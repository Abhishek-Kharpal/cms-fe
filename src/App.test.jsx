import { render } from '@testing-library/react';
import App from './App';

describe('App', () => {
  it('should render successfully', () => {
    const { asFragment } = render(<App />);
    expect(asFragment()).toMatchSnapshot();
  });
});