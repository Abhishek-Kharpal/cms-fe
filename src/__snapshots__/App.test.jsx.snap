// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render successfully 1`] = `
<DocumentFragment>
  <div>
    <div
      class="login-page"
    >
      <div
        class="entry-image"
      >
        <img
          alt="entry-image"
          src="entry-image.png"
        />
      </div>
      <div
        class="login-form-container"
      >
        <div
          class="form-header"
        >
          <div>
            Login to your CMS+ account
          </div>
        </div>
        <div
          class="form-container-1"
        >
          <form
            class="form"
          >
            <p
              style="margin: 0px;"
            >
              Email
            </p>
            <input
              name="email"
              style="margin-bottom: 10px; margin-top: 5px; height: 20px; width: 300px;"
            />
            <p
              style="margin: 0px;"
            >
              Password
            </p>
            <input
              name="password"
              style="margin-bottom: 10px; margin-top: 5px; height: 20px; width: 300px;"
              type="password"
            />
            <button
              type="submit"
            >
              Login
            </button>
            <p>
              Dont have an account? 
              <a
                href="/register"
              >
                Register
              </a>
            </p>
          </form>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
